### DISCUSSION TITLE
Export Sublists as if they were Lists

### DISCUSSION THREAD

**Discussion by amarchetanpatel** [burkybang/SuiteAdvanced-Feedback#43] [Ideas]
Is it possible to add an 'export as csv' button on sub-lists? It would be very useful to be able to export simple sub-list results into a CSV file.

Thanks!

**Comment by burkybang** [comment]
That's a cool idea. I'll give it some thought to see if it's possible.

**Comment by HBoxer** [comment]
Pulling this back up - this would be really helpful! Any plans of implementing this?
